docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -47,6 +59,23 @@ If there is an error please [contact support](mailto:pulsesupport@codacy.com).
 -   The set of changes in a deployment is the list of commits in the corresponding pull request.
 -   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
 
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The following is a detailed description of how the Pulse Bitbucket integration automatically detects incidents:
+
+### Use pull request reverts (based on default branch)
+
+-   Pulse bases incident detection on [pull request reverts](https://support.atlassian.com/bitbucket-cloud/docs/merge-a-pull-request/#Revert-a-merged-pull-request).
+-   Pulse considers an incident any pull request that **targets the default branch** of the repository merged from a branch which name starts with `revert-pr-`, getting the number of the reverted pull request from the branch name, `revert-pr-<pull request number>`. If you change the name of the branch created by Bitbucket when you revert a pull request, Pulse may not be able to obtain the incident data correctly.
+-   The incident creation date is the timestamp when the reverted pull request was initially merged. If Pulse can't get the reverted pull request number from the branch name, the incident creation date is the timestamp of the first commit to the incident pull request.
+-   Pulse associates incidents to the system matching the repository name.
+
+### Don't detect incidents via Bitbucket
+
+-   Pulse doesn't detect incidents automatically using Bitbucket events.
```

Comment:
See the [comment on the related PR](https://github.com/codacy/pulse-user-docs/pull/195/files#r1108207734). The same applies here.